ang/jsonFormatter.ang.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+// Download 10kb version or 200b version? Depends on how likely we are to need it...
+$isDebug = Civi::settings()->get('debug_enabled');
+
+return [
+  'ext' => 'civicrm',
+  'js' => [
+    $isDebug ? 'bower_components/json-formatter/dist/json-formatter.min.js' : 'ang/jsonFormatter.stub.js',
+  ],
+  'css' => [
+    $isDebug ? 'bower_components/json-formatter/dist/json-formatter.min.css' : NULL,
```

Comment:
The coupling between `$isDebug` and the ability of this thing to function feels hack-ish to me. For the purposes at hand, sure it makes sense, but what if some other module wanted this functionality for purposes other than debugging? I might even have a use for it in search kit.

Reply:
I don't want to trigger more work with that comment. This whole PR feels like a low-priority side-project and we should get back to the main tasks. If you fix the test I'll merge it as-is.

Reply:
Yeah, I can see how `jsonFormatter` could also be useful in admin/site-builder tooling.

Found a way to remove the `jsonFormatter` stub. With the update, `jsonFormatter` works like a regular module. (So if you want to use, then just... require it...) The optimization bit is now in `crmUiDebug`, which has a soft-requirement `jsonFormatter`.

Tangentially, `jsonFormatter` will also autoload now if used by an afform.